champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/knight/sword/DefensiveStance.java

```diff
@@ -152,20 +157,45 @@ public void useEnergy() {
 
             Gamer gamer = championsManager.getClientManager().search().online(player).getGamer();
             if (!gamer.isHoldingRightClick()
-                    || !championsManager.getEnergy().use(player, getName(), getEnergy(level) / 2, true)
+                    || !championsManager.getEnergy().use(player, getName(), getEnergy(level) / 20, true)
                     || (level <= 0)
                     || !isHolding(player)) {
 
                 iterator.remove();
             }
             else {
-                player.getWorld().playEffect(player.getLocation(), Effect.STEP_SOUND, 20);
+                player.getWorld().playSound(player.getLocation(), Sound.BLOCK_STONE_STEP, 0.5F, 1.0F);
+                spawnParticles(player);
             }
 
         }
 
     }
 
+    private void spawnParticles(Player player) {
```

Comment:
instead of spawning particles here, maybe use a shield set to visible(see unstoppable force)